bftengine/tests/simpleKVBCTests/TesterReplica/main.cpp

```diff
@@ -67,88 +97,80 @@ int main(int argc, char **argv) {
 		{
 			strncpy(argTempBuffer, optarg, sizeof(argTempBuffer) - 1);
 			argTempBuffer[sizeof(argTempBuffer) - 1] = 0;
-			keysFilePrefix = argTempBuffer;
+			rp.keysFilePrefix = argTempBuffer;
 			// TODO: check keysFilePrefix
 		}
 		break;
 
-		case 'f':
-		{
-			strncpy(argTempBuffer, optarg, sizeof(argTempBuffer) - 1);
-			argTempBuffer[sizeof(argTempBuffer) - 1] = 0;
-			string fStr = argTempBuffer;
-			int tempfVal = std::stoi(fStr);
-			if (tempfVal >= 1 && tempfVal < UINT16_MAX)
-				fVal = (uint16_t)tempfVal;
-			// TODO: check fVal
-		}
-		break;
-
-		case 'c':
-		{
-			strncpy(argTempBuffer, optarg, sizeof(argTempBuffer) - 1);
-			argTempBuffer[sizeof(argTempBuffer) - 1] = 0;
-			string cStr = argTempBuffer;
-			int tempcVal = std::stoi(cStr);
-			if (tempcVal >= 0 && tempcVal < UINT16_MAX)
-				cVal = (uint16_t)tempcVal;
-			// TODO: check cVal
-		}
-		break;
+		case 'n':
+        {
+          strncpy(argTempBuffer, optarg, sizeof(argTempBuffer) - 1);
+          argTempBuffer[sizeof(argTempBuffer) - 1] = 0;
+          rp.configFileName = argTempBuffer;
+        }
+        break;
 
 		default:
 			// nop
 			break;
 		}
 	}
 
-	string keysFile;
-
-	if (!keysFilePrefix.empty() && !idStr.empty())
-		keysFile = keysFilePrefix + "_" + idStr;
-
-	if(repId == UINT16_MAX || 
-		 keysFile.empty() ||
-		 fVal == UINT16_MAX ||
-		 cVal == UINT16_MAX
-		)
+	if(rp.replicaId == UINT16_MAX || rp.keysFilePrefix.empty())
 	{
-		fprintf(stderr, "%s -k KEYS_FILE_PREFIX -f F -c C -i ID", argv[0]);
+		fprintf(stderr, "%s -k KEYS_FILE_PREFIX -i ID -f COMM_CONFIG_FILE",argv[0]);

```

Comment:
typo: should be "-n COMM_CONFIG_FILE", not "-f"

Reply:
oops, thanks!